App crashes when joining your first public chat

# Bug Report

## Description

When an account has no public chats, it crashes once you do

```
DBG 2022-12-19 13:30:46.721-05:00 NewBE_callPrivateRPC                       topics="rpc" tid=31613 file=core.nim:20 rpc_method=chat_joinChat
Traceback (most recent call last)
/home/jonathan/dev/status-desktop/src/nim_status_client.nim(181) nim_status_client
/home/jonathan/dev/status-desktop/src/nim_status_client.nim(178) mainProc
/home/jonathan/dev/status-desktop/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/home/jonathan/dev/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/home/jonathan/dev/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/home/jonathan/dev/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(579) onSlotCalled
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/view.nim(145) createPublicChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/io_interface.nim(160) createPublicChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/module.nim(411) createPublicChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/controller.nim(297) createPublicChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/io_interface.nim(72) addChatIfDontExist
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/module.nim(925) addChatIfDontExist
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/module.nim(413) addNewChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/module.nim(456) addNewChat
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/chat_content/io_interface.nim(12) load
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/chat_content/module.nim(95) load
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/chat_content/input_area/io_interface.nim(11) load
/home/jonathan/dev/status-desktop/src/app/modules/main/chat_section/chat_content/input_area/module.nim(45) load
/home/jonathan/dev/status-desktop/vendor/nimqml/src/nimqml/private/qqmlapplicationengine.nim(30) setRootContextProperty
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:627: run-linux] Error 1
```

--- a/ui/main.qml
+++ b/ui/main.qml
@@ -128,7 +128,7 @@ StatusWindow {
     Connections {
         id: windowsOsNotificationsConnection
         enabled: Qt.platform.os === Constants.windows
-        target: typeof mainModule !== "undefined" ? mainModule : null
+        target: Qt.platform.os === Constants.windows && typeof mainModule !== "undefined" ? mainModule : null
         function onDisplayWindowsOsNotification(title, message) {
             systemTray.showMessage(title, message)
         }

